final/src/components/employees/EmployeeForm.js

```diff
@@ -227,8 +207,10 @@ class EmployeeForm extends Component {
           </div>
           <FormControl.Feedback />
         </FormGroup>
-
-        <Button bsStyle="success" onClick={this.handleSave} disabled={!this.validateAll()}> Save </Button>&nbsp;
+        <Button bsStyle="success" onClick={this.handleSave} disabled={!this.validateAll()}>
+          {' '}
```

Comment:
this is interesting. shouldn't whitespace collapse to a single space anyway, without the explicit `{' '}`? Or does JSX work differently than browsers/HTML?